gfx/NeoDays/pngs_tiles_10x10/monsters/feral_humans/mon_shia.json

```diff
@@ -0,0 +1,7 @@
+{
+  "id": [
+    "mon_shia",
+    "mon_zombie_jackson"
```

Comment:
those two are crazy cataclysm monsters

Reply:
They're both in standard DDA; but I'll shift things.